src/test/java/net/amygdalum/tanteemmas/RecheckTest.java

```diff
@@ -22,6 +24,15 @@
 	public void setup() {
 		vertx = Server.deployServer(8080);
 
+		// Without that, we are actually doing cross-browser testing...
+		System.setProperty(RecheckIgnore.IGNORED_ATTRIBUTES_PROPERTY, //
```

Comment:
Shouldn't we put this in `retest.properties` ([here](https://github.com/retest/tante-emmas/blob/master/src/test/resources/retest/retest.properties))? Or is it only for this test, not globally? And we don't want to use a rule for this?